lamindb/models/record.py

```diff
@@ -671,75 +671,86 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         skip_validation = kwargs.pop("_skip_validation", False)
-        if not args and skip_validation:
-            super().__init__(**kwargs)
-        elif not args and not skip_validation:
-            from ..core._settings import settings
-            from .can_curate import CanCurate
-            from .collection import Collection
-            from .schema import Schema
-            from .transform import Transform
-
-            validate_fields(self, kwargs)
-
-            # do not search for names if an id is passed; this is important
-            # e.g. when synching ids from the notebook store to lamindb
-            has_consciously_provided_uid = False
-            if "_has_consciously_provided_uid" in kwargs:
-                has_consciously_provided_uid = kwargs.pop(
-                    "_has_consciously_provided_uid"
-                )
+        if not args:
             if (
-                isinstance(self, (CanCurate, Collection, Transform))
-                and settings.creation.search_names
-                and not has_consciously_provided_uid
+                issubclass(self.__class__, Record)
+                and not self.__class__.__name__ == "Storage"
```

Comment:
Hm, then why does `Storage` have `space` at all?

Reply:
Yes, also ping me if you do the migration, because then the `Storage` table has to get its own RLS that is pretty special.

Reply:
This won't happen in the next 6-9 months. It'd happen only in LaminDB 2.0

Reply:
Ok, but we just need to keep this mind.

Reply:
Yes!